ohiecli/cmd/docker/docker.go

```diff
@@ -520,3 +506,22 @@ var untarPackage = func(tarContent io.ReadCloser) (pathToPackage string, err err
 
 	return pathToPackage, nil
 }
+
+// TODO: Check references
```

Comment:
Does this comment still need to be here?

Reply:
Remove in [70b02e8](https://github.com/openhie/package-starter-kit/pull/14/commits/70b02e8ab62dae04a74398dff39d2402b597a37f) :+1: